packages/shell-api/src/shell-internal-state.ts

```diff
@@ -352,7 +352,7 @@ export default class ShellInternalState {
         serverTypePrompt = '[primary]';
         break;
       case 'Sharded':
-        serverTypePrompt = this.connectionInfo?.extraInfo?.is_atlas ? '[atlas proxy]' : '[mongos]';
+        serverTypePrompt = this.connectionInfo?.extraInfo?.atlas_version ? '[atlas]' : '[mongos]';
```

Comment:
Just to double-check that I'm getting this right: so in the sharded case we want to explicitly check if (basically) `atlasVersion` command worked to show it as "atlas", but it's not a definitive way to check if we are communicating with atlas in general because it only works on some (probably newer) atlas servers

Reply:
Yes, exactly :+1: I guess we could also use `is_atlas` here, but since the `atlasVersion` check applies and seems a bit more reliable, I think using that is good here

Reply:
Great, thanks for confirming! Yeah, agree on using version for this case 👍